static/scss/partials/modals.scss

```diff
@@ -0,0 +1,151 @@
+.modal-message {
+    @include text-body-sm;
+    color: $color-grey-60;
+    margin: 0 auto $spacing-sm auto;
+
+    strong {
+        word-break: break-word;
+    }
+    
+    &.warning-text {
+        color: $color-error;
+    }
+}
+
+.alias-to-delete {
+    display: inline;
+    word-break: break-all;
+
+    &.demi {
+        display: inline-block;
+        margin: 0 auto $spacing-md;
+        // margin: 0 auto $spacing-md 0;
+        color: $color-grey-60;
+        max-width: $content-xs;
+        text-overflow: ellipsis;
+        overflow: hidden;
+        word-wrap: unset;
+    }
+}
+
+// Modal Protocol Refactor
+.c-modal-wrapper {
+    position: fixed;
+    display: none;
+    background: rgba(0, 0, 0, .4);
+    top: 0;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    z-index: 10;
+    justify-content: center;
+    flex-direction: column;
+
+    &.is-visible {
+        display: flex;
+    }
+}
+
+.c-modal {
+    background: $color-white;
+    max-width: $content-sm-md;
+    border-radius: $border-radius-md;
+    position: absolute;
+    left: $spacing-xl;
+    right: $spacing-xl;
+    margin: auto;
+    box-shadow: $box-shadow-sm;
+    overflow: hidden;
+    padding: $spacing-md;
+}
+
+.c-modal-header {
+    border-radius: $border-radius-md;
+    background-color: $color-grey-05;
+    padding: $spacing-lg $spacing-md;
+}
+
+.c-modal-header-content {
+    @include text-title-3xs;
+    
+    h3 {
+        @include text-title-xs;
+        color: $color-grey-40;
+        margin: $spacing-sm 0;
+        font-weight: 700;
+
+        strong {
+            color: $color-black;
+        }
+
+        span {
+            display: inline-block;
+        }
+    }
+
+    .c-modal-header-headline {
+        display: flex;
+        width: 100%;
+        align-items: center;
+        justify-content: center;
+
+        img {
+            margin-right: $spacing-sm;
+        }
+    }
+}
+
+.t-domain-registration-hero {
```

Comment:
Protocol: Is this a correct use of a `t-` class?

cc/ @craigcook

Reply:
"t" is for theming, so it's usually a modifier of some other shared class, and most often presentational. Roughly equivalent to the "M" in BEM. 

If this class is being used to visually differentiate this element from other elements of the same kind, I'd say that's an appropriate use of theming. It says "here is a box that is functionally the same as other boxes, but this particular one has a slightly different appearance." So if you have other "hero" elements but the "domain registration" hero is a different color, that would be a case for a theme class (and would probably be something like `.t-hero-domain-registration`, assuming that's how you want to describe the style and wouldn't use the same style on other heroes that aren't about domain registration).

But this class isn't getting baked into Protocol, you're just using it for this one project so you can really name it whatever makes sense to you. 🙂